Lectures/Lecture3_BigCollaborations.md

```diff
@@ -1,41 +1,62 @@
 
 ## Motivation: 
-__sketch the context of the lecture, why is this topic timely or urgent to cover, and what is desired or unwanted in the status quo.__
+Large-scale and often international collaborations are playing an increasingly important role in the contemporary research landscape in physics, as the high-end technologies and facilities required to address the driving questions require large investments and expertise. This offers unique scientific opportunities but the coordination of a large number of scientists within a collaboration while retaining individual independence also comes with a number of challenges that will be discussed. In the assignment, the credit structures in different collaborations will be examined, with particular focus on ways in which junior scientist can gain visibility within such large consortia.
 
 ## Vision of outcome
-__What skills and capabilities will the course participants obtain after finishing this lecture and the corresponding assignments?__
+- Course participants appreciate the need for and the benefits of large collaborations for many driving questions in modern science. 
+- Course participants have critically reflected on challenges faced by large collaborations and choices to be made in their structure, organization, and membership.
+- Course participants have investigated examples of working within large collaborations, in particular the credit structures and  ways to promote junior researchers. 
 
 ## Embedding/community
-__please mention the main community that should be considered when studying the topics of this lecture, e.g. individual researchers, research community, employers, general public, etc. __
+This lecture focuses on large national as well as international collaborations. Many of the large collaborations involve interdisciplinary components, both within physics and with engineering and industry. See https://bigscience.nl/about for a concerted effort in NL to foster synergies between physics and technology in large collaborations. 
 
 ## Methods/exercises
-**Topics covered in lecture**
-+ topic 1
-+ topic 2
++ In preparation for this lecture, course participants are asked to listen to the podcast and answer the quiz questions
++ In lecture, we watch the first of the two-part video lecture series by Dr Niels Martens on the epistemology of scientific collaborations. [Lecture 1 of this set](https://www.youtube.com/watch?v=M1giIxeLuIg), which is covered in class, discusses examples of 'big science' projects such as CERN and the Next Generation Event Horizon Telescope. [Lecture 2](https://www.youtube.com/watch?v=DRt1uTytWf8), which is left for students to watch on their own, focuses on ways to manage the large number of co-authors that are involved in these projects and policies for giving appropriate credit. 
```

Comment:
In the lecture (i.e. the one big room)? Or during the tutorial (i.e. the three smaller rooms)?

Reply:
corrected